switch-configuration/config/vlans.d/Hilton

```diff
@@ -0,0 +1,18 @@
+//	Conference	Center	--	VLANS	100-499
+VLAN	hiSCALE-SLOW		100	2001:470:f0fb:100::/64	10.128.128.0/21	2.4G Wireless Network in 
```

Comment:
vlan allocations for hilton look good, not that its a show stopper but any particular reason you went with 128 for the second octet for ipv4?

Reply:
That's an error... It should be 10.0... Good catch.

Reply:
I've fixed that, now 10.0.x.x.